oneanddone/users/views.py

```diff
@@ -162,7 +162,7 @@ class UserDetailAPI(generics.RetrieveUpdateDestroyAPIView):
     serializer_class = UserSerializer
 
 
-class UserListAPI(generics.ListCreateAPIView):
+class UserListAPI(generics.CreateAPIView):
     """
     API endpoint used to get a complete list of users
```

Comment:
As above, update the comment to reflect the fact that the behaviour of this class has changed.